src/adaptors/defichain-dex/index.js

```diff
@@ -0,0 +1,23 @@
+const utils = require('../utils');
+
+const API_URL = 'https://ocean.defichain.com/v0/mainnet/poolpairs?size=200';
+
+const getApy = async () => {
+  const poolsData = await utils.getData(API_URL);
+
+  const pools = poolsData.data.map((pool) => ({
+    pool: `${pool.symbol}-pool`,
```

Comment:
if we don't have addresses for these, I'd rather add the project as suffix than `-pool`